### ISSUE TITLE
Connector start failed but status always pending

### ISSUE THREAD

**Issue by vitojeng** [oharastream/ohara#4099]
啟動 Connector 時, 後端抛出錯誤, 但 UI 停留在 Penging, 且在 Event log 沒有訊息

Env: QuickstartVM
操作步驟
* Add node, Create workspace 成功
* 建立 pipeline, 建立 ftp source connector, ftp sink connector, 建立 source/sink 連線
* 在 ftp source connector 填入 settings, 然後按下 Start
* ftp source connector 顯示橘燈(Pending), 然後一直在橘燈狀態
  * 實際上 API 拋出錯誤
  * Even log 沒有訊息

 > Always at "Penging" state

![screenshot 22](https://user-images.githubusercontent.com/475186/75125514-71d2c680-56f0-11ea-9161-80e15744b516.jpg)

> Browser develop tool

![screenshot 23](https://user-images.githubusercontent.com/475186/75125526-82833c80-56f0-11ea-98a7-bf97dec622dc.jpg)

> Actual error from rest api
> Request URL: http://192.168.56.109:5050/api/connectors/ftpsource1?group=fcfb6f54c

```
error: oharastream.ohara.common.exception.OharaFileSystemException: Could not parse response code.
Server Reply: SSH-2.0-OpenSSH_7.4
	at oharastream.ohara.client.filesystem.FileSystem.wrap(FileSystem.scala:102)
	at oharastream.ohara.client.filesystem.FileSystem.wrap$(FileSystem.scala:98)
	at oharastream.ohara.client.filesystem.ftp.FtpFileSystem$Builder$FtpFileSystemImpl.wrap(FtpFileSystem.scala:135)
	at oharastream.ohara.client.filesystem.ftp.FtpFileSystem$Builder$FtpFileSystemImpl.exists(FtpFileSystem.scala:142)
	at oharastream.ohara.client.filesystem.FileSystem.nonExists(FileSystem.scala:51)
	at oharastream.ohara.client.filesystem.FileSystem.nonExists$(FileSystem.scala:51)
	at oharastream.ohara.client.filesystem.ftp.FtpFileSystem$Builder$FtpFileSystemImpl.nonExists(FtpFileSystem.scala:135)
	at oharastream.ohara.connector.ftp.FtpSource._start(FtpSource.scala:48)
	at oharastream.ohara.kafka.connector.RowSourceConnector.start(RowSourceConnector.java:120)
	at org.apache.kafka.connect.runtime.WorkerConnector.doStart(WorkerConnector.java:110)
	at org.apache.kafka.connect.runtime.WorkerConnector.start(WorkerConnector.java:135)
	at org.apache.kafka.connect.runtime.WorkerConnector.transitionTo(WorkerConnector.java:195)
	at org.apache.kafka.connect.runtime.Worker.startConnector(Worker.java:257)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder.startConnector(DistributedHerder.java:1183)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder.access$1400(DistributedHerder.java:125)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder$15.call(DistributedHerder.java:1199)
	at org.apache.kafka.connect.runtime.distributed.DistributedHerder$15.call(DistributedHerder.java:1195)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.commons.net.MalformedServerReplyException: Could not parse response code.
Server Reply: SSH-2.0-OpenSSH_7.4
	at org.apache.commons.net.ftp.FTP.__getReply(FTP.java:344)
	at org.apache.commons.net.ftp.FTP.__getReply(FTP.java:300)
	at org.apache.commons.net.ftp.FTP._connectAction_(FTP.java:438)
	at org.apache.commons.net.ftp.FTPClient._connectAction_(FTPClient.java:962)
	at org.apache.commons.net.ftp.FTPClient._connectAction_(FTPClient.java:950)
	at org.apache.commons.net.SocketClient._connect(SocketClient.java:244)
	at org.apache.commons.net.SocketClient.connect(SocketClient.java:202)
	at oharastream.ohara.client.filesystem.ftp.FtpClient$Builder$FtpClientImpl.connectIfNeeded(FtpClient.scala:318)
	at oharastream.ohara.client.filesystem.ftp.FtpClient$Builder$FtpClientImpl.exist(FtpClient.scala:476)
	at oharastream.ohara.client.filesystem.ftp.FtpClient$Builder$$anon$1.$anonfun$exist$1(FtpClient.scala:303)
	at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.java:23)
	at oharastream.ohara.client.filesystem.ftp.FtpClient$Builder$$anon$1.retry(FtpClient.scala:281)
	at oharastream.ohara.client.filesystem.ftp.FtpClient$Builder$$anon$1.exist(FtpClient.scala:303)
	at oharastream.ohara.client.filesystem.ftp.FtpFileSystem$Builder$FtpFileSystemImpl.$anonfun$exists$1(FtpFileSystem.scala:143)
	at scala.runtime.java8.JFunction0$mcZ$sp.apply(JFunction0$mcZ$sp.java:23)
	at oharastream.ohara.client.filesystem.FileSystem.wrap(FileSystem.scala:100)
	... 20 more
```

- [x] for 0.10 (move to #4131)
- [x] backport to 0.9 (#4141)

**Comment by vitojeng** [comment]
以下可能是類似的情形, 操作步驟:
* 接續上方的情形, refresh page, Connector 呈現紅色狀態(failed), 此時 connector button enabled.
* 按下 Stop 後(此時 connector button disabled)
* 後端抛出錯誤, connector 停留在 failed 狀態, 此時 button 維持在 disabled 無法再進行任何操作.

![screenshot 24](https://user-images.githubusercontent.com/475186/75126931-27087d00-56f7-11ea-996d-e8dcc826a9a9.jpg)

**Comment by saivirtue** [comment]
Root cause is the exception was happened but not raise to the UI...We will fix this issue for the current version in #4131, but use a workaround for 0.9 because of resource concern.

**Comment by chia7712** [comment]
Great!
